src/App.jsx

```diff
@@ -3,26 +3,87 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { request } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    titleSearch: '',
+    todosStatus: 'All',
   };
 
+  componentDidMount() {
+    request('/todos')
+      .then(({ data }) => 
+        this.setState({ todos: [...data] })
+      );
+  }
+
+  selectUser = (userId) => {
+    if (this.state.selectedUserId !== userId) {
+      this.setState({ selectedUserId: userId });
+    }
+  }
+
+  clearUser = () => {
+    this.setState({ selectedUserId: 0 });
+  }
+
+  titleSearchChangeHandler = ({ name, value }) => {
+    this.setState({ [name]: value });
+  }
+
+  setTodosStatus = (status) => {
+    this.setState({ todosStatus: status });
+  }
+
+  prepareTodos = () => {
```

Comment:
You can create list prepareTodos in render without method.

Reply:
It is strongly recommended not to overload render method with logic, so this way is correct.

Reply:
Okay, sorry for missadvice